src/QsCompiler/QirGeneration/Subtransformations/ExpressionKindTransformation.cs

```diff
@@ -1199,7 +1157,10 @@ public override ResolvedExpressionKind OnConditionalExpression(TypedExpression c
             }
             else
             {
-                var evaluated = this.ConditionalEvaluation(cond.Value, onCondTrue: ifTrueEx, onCondFalse: ifFalseEx);
+                var evaluated = this.SharedState.ConditionalEvaluation(
+                    cond.Value,
+                    onCondTrue: () => this.SharedState.EvaluateSubexpression(ifTrueEx),
+                    onCondFalse: () => this.SharedState.EvaluateSubexpression(ifFalseEx));
```

Comment:
nit: I prefer aligning the parts of the similar strings such that it is immediately visible what's the difference between the two strings, like:
```c#
                    onCondTrue : () => this.SharedState.EvaluateSubexpression(ifTrueEx ),
                    onCondFalse: () => this.SharedState.EvaluateSubexpression(ifFalseEx));
```

Reply:
Personally I also like that very much. However, it is not compatible with the C# formatting rules that are enforced (or the F# ones for that matter...).